wagn/lib/wagn/generators/wagn/templates/config/routes.erb

```diff
@@ -1,6 +1,6 @@
 # -*- encoding : utf-8 -*-
 
-require 'decko/engine'
+#require 'decko/engine'
```

Comment:
I'm going to test this use case further on the generator branch.  It shouldn't be used in the *app deck* case anyway.  I think it is better if we don't need this, and this require should probably happen automatically via whatever is auto-included.

I don't know if you noticed, but I took away some of the eager load options in the config (as part of the initial card-gem split).  Best practice is not to do that for lib directories.